replication/row_event.go

```diff
@@ -985,6 +994,28 @@ func (e *RowsEvent) DecodeHeader(data []byte) (int, error) {
 	return pos, nil
 }
 
+func (e *RowsEvent) decodeExtraData(pos int, data []byte) (err2 error) {
+	extraDataType := data[pos]
+	pos += 1
+	switch extraDataType {
+	case 0:
```

Comment:
it's better to add a const for `0`, like ENUM_EXTRA_ROW_INFO_TYPECODE_NDB or something. Same for below `1` case.